xmppserver/src/main/java/org/jivesoftware/openfire/muc/spi/MUCPersistenceManager.java

```diff
@@ -499,9 +505,17 @@ public static void deleteFromDB(MUCRoom room) {
         ResultSet resultSet = null;
         try {
             connection = DbConnectionManager.getConnection();
-            statement = connection.prepareStatement(LOAD_ALL_ROOMS);
-            statement.setLong(1, serviceID);
-            statement.setString(2, StringUtils.dateToMillis(emptyDate));
+            if (emptyDate!=null) 
```

Comment:
This was already in the code, but it confused me. Can you improve this? The argument that is named `emptyDate` really is not the "empty date" - it is the cleanup date. Can you rename the argument please (and do the same in the method that calls this?

Reply:
i think its called emptyDate here because the flag in the DB is also called emptyDate which is asked in the SQL Query but i've changed it